src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -43,99 +52,141 @@
 public class AddCommandParserTest {
     private AddCommandParser parser = new AddCommandParser();
 
+    /**
```

Comment:
I think is great that you added the javadoc and comments for this test cases to indicate what we are testing! :+1: